src/_locales/ja/messages.json

```diff
@@ -140,7 +140,7 @@
     "person_or_group_in_list_many": "{{people}}… 全部で{{count}}",
     "person_or_group_in_list_many_no_preview": "全部で{{count}}",
     "personas": "人格",
-    "please_include_proof_your_bio": "以下の「 」内の内容を自己紹介欄に追加すれば Maskbook が使えるようになります。「{{bio}}」",
+    "please_include_proof_your_bio": "「{{bio}}」の中身をコピーし、プロフィールの自己紹介欄に追加してしてください。でなければ Maskbook を正常に使用できない場合があります。",
     "popup_current_persona": "現在の人格アカウント",
     "popup_current_wallet": "",
```

Comment:
the is empty.

Reply:
Sorry what does "the is empty." mean?

Reply:
Please help us fill in the content, thank

Reply:
Please search `""` string.